packages/jest-runtime/src/index.js

```diff
@@ -433,8 +433,8 @@ class Runtime {
     }
   }
 
-  getAllCoverageInfo() {
-    return this._environment.global.__coverage__;
+  getAllCoverageInfoCopy() {
+    return JSON.parse(JSON.stringify(this._environment.global.__coverage__));
```

Comment:
This is probably the _worst_ way to do this but I figured I would push and ask if there's a preferred way to deep copy objects in this project. I tried `deepCyclicCopy` from `jest-utils` but that did not work

Reply:
For the curious, this is what the object looks like: https://gist.github.com/rickhanlonii/337bb5310f827cef35ec2d329edbfc3b

Reply:
@rickhanlonii What was the issue with `deepCyclicCopy`? It's true it's too basic to copy `RegExp`, `Date`s and such, but being JSON copy-able, it should work out of the box 😔

Reply:
Checking at the code, what can be happening is that we create the copied object using the prototype of the original one, so the `instanceof` operator still works for the copied one (this becomes especially important when creating copies of objects like `process`).

But, in turn, it can be preventing the memory to release, which I guess it's why it did not work for you.

Reply:
Yeah that makes sense, that is what's happening